config/api.yml

```diff
@@ -2211,6 +2211,24 @@
       :get:
       - :name: read
         :identifier: miq_report_view
+  :search_filters:
+    :description: Filters
+    :verbs: *gpd
```

Comment:
with get enabled, we should have a test added in collections_spec.rb to test them.